login/ConfigConnection.json

```diff
@@ -0,0 +1,5 @@
+{
+  "database": "web",
+  "user": "root",
+  "password": "Edding16"
```

Comment:
I don't think this a safe way to store passwords, really easy for someone to find this password out if they're just browsing through your repo, maybe you should research on how to store passwords as environment variables or something of the sort in node.

Reply:
i will read about it,thx